alacritty_terminal/Cargo.toml

```diff
@@ -14,7 +14,7 @@ bitflags = "1"
 parking_lot = "0.10.2"
 serde = { version = "1", features = ["derive"] }
 serde_yaml = "0.8"
-vte = { version = "0.8.0", default-features = false }
+vte = { git = "https://github.com/alacritty/vte", default-features = false }
```

Comment:
I'd merge this against the git branch for now, then we can test the new library ourselves before making a release.

Reply:
I don't like having git version here in general. Maybe we should create a vte beta release, I guess it'll make more sense?

Reply:
Good point, this is going to make bisecting in the future impossible, I didn't think about that. If we use a git branch we need to specify the revision.

I feel like pinning to a commit is probably better than just uploading every random commit to crates.io though? Assuming there might be other changes to vte.

I think it might be a nice thing to think about in general, with stuff like crossfont too.

Reply:
Assuming that commit won't go away, then maybe it's fine to do so, but using versions from crates.io are more safe in general.

Reply:
I mean commits should never go away, so I think that's pretty safe.

I just don't want to get into a situation where every time we want to test a development version of one of our crates, we need to start jumping through hoops and release new versions.

If we instead just linked to a specific commit, that should allow us to implement new stuff without having to put in any work.

But of course crates.io is less likely to go away than our github repository.

Reply:
let's stick to specific hash then and try how it works.